portal/templates/redesign/base_new.html

```diff
@@ -13,7 +13,7 @@
 
     <title>{% block title %}Code for Life{% endblock %}</title>
     <meta name="msapplication-config" content="none"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
End slash has been removed from this meta, but not the previous. The end slash is optional, but we should be consistent in style. At least in the same file.